service-api/module/Application/config/module.config.php

```diff
@@ -206,6 +208,46 @@
                     ],
                 ],
             ],
+            'find_postoffice_branches' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/counter-service/branches',
+                    'defaults' => [
+                        'controller' => Controller\YotiController::class,
+                        'action'     => 'findPostOffice',
+                    ],
+                ],
+            ],
+            'create_yoti_session' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/counter-service/create-session',
+                    'defaults' => [
+                        'controller' => Controller\YotiController::class,
+                        'action'     => 'createSession',
+                    ],
+                ],
+            ],
+            'retrieve_yoti_status' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/counter-service[/:sessionId]/retrieve-status',
```

Comment:
This URL should accept a case UUID, retrieve the session ID from its CaseData, and use that in the Yoti API. Otherwise the frontend would need to be aware of the session ID, but the frontend is stateless.
```suggestion
                    'route'    => '/counter-service/:uuid/retrieve-status',
```